app/controllers/npc_pdfs_controller.rb

```diff
@@ -0,0 +1,20 @@
+class NpcPdfsController < ApplicationController
+  def show
+    @npc = NpcModel.where(id: params[:id])[0]
+    @split_spells_slots = @npc.spell_slots.split('{')[1].split('}')[0].split(',').map { |x| x[-1] } unless nil
```

Comment:
This line potentially won't be necessary if we end up making our own PDF instead of using the template I've currently got set as the show.